### PR TITLE
aestetic changes and minor html modifications

### PR DESCRIPTION
**Pull request by belen-albeza**
Basically added more white space (padding, margins, etc) all around. Also removed the italics in the FAQ page since it makes it so hard to read.

I don't know how to push my fork to github pages so you can have a full preview, so a screenshot will have to do:

screen shot 2016-01-07 at 18 47 33

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -219,9 +233,10 @@ footer a:hover {
   margin: 0;
 }
 
-.faq p,
-.resources p {
-  margin: 0.2rem 0 0.5rem 0;
+.faq p:last-child,
+.resources p:last-child {
+  /*margin: 0.2rem 0 0.5rem 0;*/
```
Comment (delapuente) [inline css/style.css]: Why are we keeping this?

Comment (belen-albeza) [inline css/style.css]: Because I forgot to remove that line :blush:

Comment (delapuente) [inline css/style.css]: Remove the line and we can merge. Thank you!

**Comment by belen-albeza** [comment]
Removed the line that was commented by mistake

**Pull request merged**